doc/apimanager-reference.md

```diff
@@ -326,3 +326,15 @@ existing secret names.
 | --- | --- | --- |
 | AWS_ACCESS_KEY_ID | AWS Access Key ID to use in S3 Storage for System's file storage | N/A |
 | AWS_SECRET_ACCESS_KEY | AWS Access Key Secret to use in S3 Storage for System's file storage | N/A |
+
+#### system-smtp
+
+| **Field** | **Description** | **Default value** |
+| --- | --- | --- |
+| address | Address (hostname or IP) of the remote mail server to use. If set to a value different than `""` System will use the mail server to send mails related to events that happen in the API management solution |  `""` |
+| username | In case the mail server requires authentication, set the username using this setting | `""` |
+| password | In case the mail server requires authentication, set the password using this setting | `""` |
+| domain | If a HELO domain is needed to be specified set it using this setting | `""` |
+| port | Port of the remote mail server to use | `""` |
+| authentication | In case the mail server requires authentication, set this setting to the authentication type here. `plain` to send the password in the clear, `login` to send password Base64 encoded, or `cram_md5` to combine a Challenge/Response mechanism to exchange information and a cryptographic Message Digest 5 algorithm to hash important information | `""` |
+| openssl.verify.mode | When using TLS, you can set how OpenSSL checks the certificate. This is really useful if you need to validate a self-signed and/or a wildcard certificate. You can use the name of an OpenSSL verify constant: `none` or `peer` | `""` |
```

Comment:
```suggestion
| openssl.verify.mode | When using TLS, use this field to specify how OpenSSL checks the certificate. This is useful to validate a self-signed and/or a wildcard certificate. Use the name of an OpenSSL verify constant: `none` or `peer` | `""` |
```

Reply:
<img width="181" alt="Screenshot 2019-12-18 18 42 19" src="https://user-images.githubusercontent.com/54224/71109738-29fe6300-21c6-11ea-8b77-13ece463bac2.png">

wouldn't it make sense to re-order and rename the fields to:
- address
- port
- helo.domain
- auth.type
- auth.username
- auth.password
- openssl.verify.mode

Reply:
Ok to the order.
You cannot rename the fields as they are the names that System looks.